conf/application.conf

```diff
@@ -79,4 +79,5 @@ feature {
      userAllowListEnabled = false
```

Comment:
Can we remove drop_1_1_enabled as this is not used any more?

Reply:
Removed